packages/abi-gen-wrappers/src/generated-wrappers/asset_proxy_owner.ts

```diff
@@ -204,7 +172,11 @@ export class AssetProxyOwnerContract extends BaseContract {
          * @param txData Additional data for transaction
          * @returns The hash of the transaction
          */
-        async sendTransactionAsync(owner: string, txData?: Partial<TxData> | undefined): Promise<string> {
+        async sendTransactionAsync(
+            owner: string,
+            txData?: Partial<TxData> | undefined,
+            opts: TxOpts = { shouldValidate: true },
```

Comment:
This was also the default behaviour a few months back before we started the whole multi-lang/abi-gen project. Neither @feuGeneA nor myself understood the importance of performing an `eth_call` first so I treated it as an optional step! But have since learned that it's a good idea to always validate before sending a tx.